core/services/eth/node.go

```diff
@@ -62,8 +65,7 @@ type node struct {
 func NewNode(lggr logger.Logger, wsuri url.URL, httpuri *url.URL, name string) Node {
 	n := new(node)
 	n.name = name
-	n.log = lggr.With(
-		"nodeName", name,
+	n.log = lggr.Named("Node").Named(name).With(
```

Comment:
Does this make sense? Or should `name` stay a separate key/val?

Reply:
Do the names stack? How does it look in the logging output?

Reply:
Yeah, dot-separated suffixes. e.g:
##### Run
```
logger=Node.eth-primary-0 

logger=SendOnlyNode.eth-sendonly-0

logger=BulletproofTxManager.bptxm_reaper

logger=BulletproofTxManager.EthResender 

logger=BulletproofTxManager.block_history_estimator 

logger=BulletproofTxManager.EthConfirmer
```
##### Test
```
logger=TestEthClient_BalanceAt/larger_than_signed_64_bit_integer.Node.eth-primary-0 

logger=TestEthClient_SendTransaction_WithSecondaryURLs.SendOnlyNode.eth-sendonly-0

logger=TestClient_ImportExportETHKey_WithChains.BulletproofTxManager.bptxm_reaper

logger=TestClient_ConfigureChain.BulletproofTxManager.EthResender 

logger=TestClient_ConfigureChain.BulletproofTxManager.block_history_estimator 

logger=TestClient_ImportExportETHKey_WithChains.BulletproofTxManager.EthConfirmer
```